README.md

```diff
@@ -11,13 +11,15 @@ Let's load up the library and set a logging level.
 And we'll define a fixed timestamp to use so the output here in the README is
 stable over time.
 ```ocaml
-# let tags = Logs.Tag.empty |> Logs.Tag.add Ezlogs_cli.Common_tags.epoch 2590779494.386;;
+# let timestamp = Ptime.of_float_s 2590779494.386 |> Option.get;;
```

Comment:
MDX? Nice 💯 
Are these functions directly accessible via the API to the caller?

Reply:
I'm starting to wonder if these tests can be autogenerated from a ppx 🤔 💭

Reply:
The more I'm using it, the more I'm a fan of mdx.  It's really really nice.

Regarding function availability - yes, they're available directly from the API.

For the ppx question - yes, we can and eventually should include/write a ppx to make this as painless as possible!  But I don't want to go down that path until we have the underlying API relatively stable so that the ppx can be relatively stable too.

In the specific case of the timestamp, the library will add a timestamp for you if one doesn't exist.  The README doesn't take advantage of that in order to keep the rendered mdx output consistent over time.